Grids AI Assistant: Add A11y Exceptions (#8845)
DevExpress/devextreme-documentation: concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md

    <tr>
        <td>501 (Web)(Software) <br> 504.2 (Authoring Tool) <br> 602.3 (Support Docs)</td>
        <td>1.1.1 Non-text Content (Level A)</td>
        <td>
            <ul>

>                 <li>Drag icons do not have <code>alt</code> and <code>aria-label</code> attributes.</li>

Reviewer: ```suggestion
                <li>Drag icons do not have <code>alt</code> or <code>aria-label</code> attributes.</li>
```

>                 <li>AI Assistant: Message status icons and command-result indicators have no <code>alt</code>/<code>aria-label</code> attributes.</li>

Reviewer: ```suggestion
                <li>AI Assistant: Message status icons and command-result indicators do not have <code>alt</code> or <code>aria-label</code> attributes.</li>
```

        <td>
            <ul>
                <li>ColumnChooser items in 'drag' mode do not support keyboard navigation.</li>
                <li>Users cannot resize columns using the keyboard (when <a href="/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#allowColumnResizing">allowColumnResizing</a> is enabled).</li>

>                 <li>AI Assistant: The "Regenerate" button is not focusable and cannot be clicked using a keyboard.</li>

Reviewer: ```suggestion
                <li>AI Assistant: The "Regenerate" button is not keyboard accessible.</li>
```

You cannot `click` something with the keyboard

            </ul>
        </td>
    </tr>
    <tr>
        <td>501 (Web)(Software) <br> 504.2 (Authoring Tool) <br> 602.3 (Support Docs)</td>
        <td>2.4.3 Focus Order (Level A)</td>
        <td>
            <ul>

>                 <li>AI Assistant: Focus is not returned to the toolbar button after the popup is closed.</li>

Reviewer: ```suggestion
                <li>AI Assistant: Focus does not return to the toolbar button after the pop-up is closed.</li>
```

>                 <li>AI Assistant: Focus is not returned to the chat input after a request is completed.</li>

Reviewer: ```suggestion
                <li>AI Assistant: Focus does not return to the chat input after a request is completed.</li>
```
